src/cloud/digitalocean/provisioner.ts

```diff
@@ -104,8 +105,13 @@ class Provisioner {
    * @param {String} name of VM to create
    * @return {Promise.<Object>}
    */
-  public stop = (name: string) : Promise<Object> => {
-    return this.doRequest_('GET', 'droplets').then((resp: any) => {
+  public stop = (name: string): Promise<Object> => {
+    this.sendStatus_('STOP');
+    return this.doOAuth_().then((oauthObj: any) => {
```

Comment:
It appears that using Object does not compile because you can't call `.droplets` on type Object. I assume this is why `resp :any` has been used throughout provisioner.ts. Any suggestions?